src/Domain/LeanCode.DomainModels.MassTransitRelay/Outbox/PeriodicEventsPublisher.cs

```diff
@@ -36,6 +39,8 @@ public class PeriodicEventsPublisher : IPeriodicAction
 
         public async Task ExecuteAsync(CancellationToken stoppingToken)
         {
+            using var fetchActivity = LeanCodeActivitySource.Start("outbox.fetch-unpublished");
```

Comment:
On the other hand, activity start could be moved to `PeriodicHostedService` to have it covered once and for all for all actions.